### PR TITLE
fix(deps-dev): address serialize-js vulnerability

### PR DESCRIPTION
**Pull request by cka121**
### My PR contains...

- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [ ] Bug fixes (non-breaking change which fixes an issue)
- [ ] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [ ] are not breaking changes.

### Documentation
- [ ] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [ ] My changes can not or do not need to be tested.
- [ ] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,6 +46,9 @@
   "workspaces": [
     "./packages/*"
   ],
+  "overrides": {
+    "serialize-javascript": "^7.0.5"
```
Comment (robert-hebel-sb) [inline package.json:50]: good enough for now
imo we could remove it if we migrate to Vitest, as we already have Vite configured in the project

https://vitest.dev/guide/migration.html#mocha-chai-sinon

**Review by robert-hebel-sb — APPROVED** [review]
